cli/azd/test/functional/aspire_test.go

```diff
@@ -235,9 +235,6 @@ func Test_CLI_Aspire_DetectGen(t *testing.T) {
 
 // Test_CLI_Aspire_Deploy tests the full deployment of an Aspire project.
 func Test_CLI_Aspire_Deploy(t *testing.T) {
-	if cfg.CI && os.Getenv("AZURE_RECORD_MODE") != "live" {
-		t.Skip("skipping live test")
-	}
```

Comment:
@weikanglim do we not have recordings here anymore?

Reply:
We never had recordings for this, was slightly worried about overall flakiness/usefulness